docs/UndoRedo.md

```diff
@@ -0,0 +1,184 @@
+# Undo/Redo
+
+The undo/redo functionality allows the user to revert changes made while editing the dashboard. 
+
+## Properties
+
+### `State`
+
+The dashboard state stores all necessary information to render dashboard. 
+
+Type: Object
+
+A state contains the following fields:
```

Comment:
Probably don't need the whole dashboard state fields documented here, only the ones related to Undo/Redo. `stretchToFit` seems like it'd be unrelated as well as a couple others.